src/frequenz/sdk/actor/power_distributing/power_distributing.py

```diff
@@ -32,28 +30,23 @@
 
 class PowerDistributingActor(Actor):
     # pylint: disable=too-many-instance-attributes
-    """Actor to distribute the power between batteries in a microgrid.
+    """Actor to distribute the power between components in a microgrid.
 
-    The purpose of this tool is to keep an equal SoC level in all batteries.
-    The PowerDistributingActor can have many concurrent users which at this time
-    need to be known at construction time.
+    The actor can have many concurrent component managers.
+    For each component manager a bidirectional channel needs to be created
+    through which they can send and receive requests and responses.
 
-    For each user a bidirectional channel needs to be created through which
-    they can send and receive requests and responses.
-
-    It is recommended to wait for PowerDistributingActor output with timeout. Otherwise if
-    the processing function fails then the response will never come.
+    It is recommended to wait for PowerDistributingActor output with timeout.
+    Otherwise if the processing function fails then the response will never come.
     The timeout should be Result:request_timeout + time for processing the request.
 
     Edge cases:
-    * If there are 2 requests to be processed for the same subset of batteries, then
-    only the latest request will be processed. Older request will be ignored. User with
-    older request will get response with Result.Status.IGNORED.
-
-    * If there are 2 requests and their subset of batteries is different but they
-    overlap (they have at least one common battery), then then both batteries
-    will be processed. However it is not expected so the proper error log will be
-    printed.
+    * A new power request will be ignored if a power request with the same
+    components is currently being processed.
+
+    * If there are 2 requests and their components is different but they overlap
+    (they have at least one common component), then both requests will be
+    processed.
```

Comment:
The power manager will make sure this doesn't happen, but I think we should ensure there are no overlaps, else requests should be queued up (just the latest, discarding older queue items).

But creating an issue and handling later is fine, because overlapping component IDs are not possible at the moment, with the power manager.

Reply:
https://github.com/frequenz-floss/frequenz-sdk-python/issues/1030